apps/passport-server/src/services/telegramService.ts

```diff
@@ -577,28 +454,11 @@ export class TelegramService {
       process.env.SERVER_EDDSA_PRIVATE_KEY
     );
 
-    if (isLocalServer()) {
-      eventIdMatch = true;
-      signerMatch =
-        pcd.claim.signer[0] === TICKETING_PUBKEY[0] &&
-        pcd.claim.signer[1] === TICKETING_PUBKEY[1];
-    } else if (process.env.PASSPORT_SERVER_URL?.includes("staging")) {
-      eventIdMatch = eventIdsAreValid(pcd.claim.validEventIds);
-      signerMatch =
-        pcd.claim.signer[0] === TICKETING_PUBKEY[0] &&
-        pcd.claim.signer[1] === TICKETING_PUBKEY[1];
-    } else {
-      eventIdMatch = eventIdsAreValid(pcd.claim.validEventIds);
-      signerMatch =
-        pcd.claim.signer[0] === TICKETING_PUBKEY[0] &&
-        pcd.claim.signer[1] === TICKETING_PUBKEY[1];
-    }
```

Comment:
We don't need separate logic now that eventIds and signers are derived, rather than hardcoded!